### ISSUE TITLE
Allow annotation to be resized after rendering

### ISSUE THREAD

**Issue by pawelfus** [blacklabel/annotations#13]
Probably 4 or 8 control points would be suitable to allow resize x/y/width/height.

**Comment by gopaldigient** [comment]
can i  get  examples for annotations with resizing enabled????

**Comment by pawelfus** [comment]
Unfortunately, this is not supported yet. How would you like to see effect of resizing for example "text" annotation?

**Comment by gopaldigient** [comment]
Thanks for the reply,rectangles that can be resized from  all the sides i:e 4 control points would be great if added.

**Comment by pawelfus** [comment]
That is correct, my question is rather what you would expect to see when resizing `Text` annotation? Or maybe resizing should be possible only for rect/line/path.

Also, what about custom annotation (not the one comes out of the box in the plugin) - how those should be managed? For example one can have "arrow" annotation - how to add control points there? 

Your suggestions are really appreciated!

**Comment by gopaldigient** [comment]
Resizing only for rect/line/path would be sufficient in case of resizing .In case of resizing texts and custom shapes and arrows a predefined or fixed resizing equally from all sides would be good.let the shapes alone have custom resizing by  the user from all the ends.

**Comment by gopaldigient** [comment]
![chart_box](https://cloud.githubusercontent.com/assets/15324435/13603436/3d21332a-e563-11e5-8694-cc6531ebf4d0.png)
this type of resizing would be good.

**Comment by Izothep** [comment]
+1
